File: src/main/java/com/likelionknu/applyserver/application/data/repository/RecruitAnswerRepository.java
package com.likelionknu.applyserver.application.data.repository;

import com.likelionknu.applyserver.application.data.entity.Application;
import com.likelionknu.applyserver.application.data.entity.RecruitAnswer;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface RecruitAnswerRepository extends JpaRepository<RecruitAnswer, Long> {
    List<RecruitAnswer> findByApplication(Application application);
    List<RecruitAnswer> findAllByApplication(Application application);
    List<RecruitAnswer> findByApplicationId(Long applicationId);
    void deleteByApplication_Id(Long applicationId);
}

File: src/main/kotlin/com/likelionknu/applyserver/application/data/repository/RecruitAnswerRepository.kt
package com.likelionknu.applyserver.application.data.repository

import com.likelionknu.applyserver.application.data.entity.Application
import com.likelionknu.applyserver.application.data.entity.RecruitAnswer
import org.springframework.data.jpa.repository.JpaRepository

interface RecruitAnswerRepository : JpaRepository<RecruitAnswer, Long> {
    fun findByApplication(application: Application): List<RecruitAnswer>
    fun findAllByApplication(application: Application): List<RecruitAnswer>
    fun findByApplicationId(applicationId: Long): List<RecruitAnswer>
    fun deleteByApplication_Id(applicationId: Long)
}